src/app/pages/component-viewer/component-examples.html

```diff
@@ -4,5 +4,7 @@
   </span>
   <example-viewer
     *ngFor="let example of docItem?.examples"
-    [example]="example"></example-viewer>
+    [example]="example"
+    [showCompactToggle]="false"
```

Comment:
Is this also an initial value? If so, I think it might have the same potential for overwriting changes as `initialView` did.